README.md

```diff
@@ -1240,7 +1241,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Swapnil Mane](https://github.com/SwapnilMane141)
   - [Swapnil Shekhar](https://github.com/Swapnil-programmer7)
   - [Syed Sibteali Baqar](https://github.com/sibteali786)
-  - [syedareehaquasar](https://github.com/syedareehaquasar)
+  - [syedareehaquasar](https://github.com/syedareehaquasar)  
```

Comment:
Trailing whitespace in here.
```suggestion
  - [syedareehaquasar](https://github.com/syedareehaquasar)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient